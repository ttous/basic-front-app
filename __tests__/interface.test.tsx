import * as React from 'react';
import * as renderer from 'react-test-renderer';

import About from '../src/scripts/pages/About';
import Home from '../src/scripts/pages/Home';

test('About', () => {
    const tree = renderer
        .create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
});

test('Home', () => {
    const tree = renderer
        .create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
});
