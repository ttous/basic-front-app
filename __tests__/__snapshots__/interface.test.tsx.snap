// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About 1`] = `
<div>
  <h1>
    About us :-)
  </h1>
</div>
`;

exports[`Home 1`] = `
<div>
  <h1>
    Welcome!
  </h1>
  <div
    id="sample-image"
  />
</div>
`;
